cub/test/CMakeLists.txt

```diff
@@ -348,31 +362,33 @@ foreach (test_src IN LISTS test_srcs)
         list(APPEND defs VAR_IDX=${var_idx})
 
         # Check if the test has explicit CDP variants:
-        _cub_has_cdp_variant(explicit_cdp "${label}")
-        _cub_is_cdp_enabled_variant(enable_cdp "${label}")
+        _cub_has_lid_variant(explicit_launcher "${label}")
+        _cub_launcher_id(explicit_launcher_id "${label}")
+
+        if (${explicit_launcher})
+          set(launcher_id "${explicit_launcher_id}")
+        else()
+          if (${CUB_FORCE_RDC})
+            set(launcher_id 1)
+          else()
+            set(launcher_id 0)
+          endif()
+        endif()
+
+        cub_cdp_by_lid(cdp_val "${launcher_id}")
 
-        if (enable_cdp AND NOT CUB_ENABLE_RDC_TESTS)
+        if (cdp_val AND NOT CUB_ENABLE_RDC_TESTS)
           continue()
         endif()
 
         # Enable RDC if the test either:
-        # 1. Explicitly requests it (cdp_1 label)
-        # 2. Does not have an explicit CDP variant (no cdp_0 or cdp_1) but
+        # 1. Explicitly requests it (lid_1 label)
+        # 2. Does not have an explicit CDP variant (no lid_0, lid_1, or lid_2) but
         #    RDC testing is forced
         #
-        # Tests that explicitly request no cdp (cdp_0 label) should never enable
+        # Tests that explicitly request no cdp (lid_0 label) should never enable
```

Comment:
What about graph capture + RDC? Is device-side graph capture supported, and if so, should we have coverage?